src/Microsoft.ML.StandardLearners/Standard/Online/OnlineLinear.cs

```diff
@@ -19,27 +19,41 @@ namespace Microsoft.ML.Trainers.Online
 
     public abstract class OnlineLinearArguments : LearnerInputBaseWithLabel
```

Comment:
>OnlineLinearArguments [](start = 26, length = 21)

xml

Reply:
added

---
In reply to: [256251874](https://github.com/dotnet/machinelearning/pull/2517#discussion_r256251874) [](ancestors = 256251874)